ext/node/polyfills/http.ts

```diff
@@ -770,6 +770,10 @@ class ClientRequest extends OutgoingMessage {
     }
     this.destroyed = true;
 
+    const rid = this._client?.rid;
+    if (rid) {
+      core.tryClose(rid);
+    }
```

Comment:
@crowlKats can we somehow abort pending request here or we don't have a machinery for that yet?

Reply:
it should be possible to use a cancel handle just like do for fetch & abortsignal